src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -680,7 +687,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         elementNames = ArrayBuilder<string>.GetInstance(arguments.Count);
                         for (int j = 0; j < i; j++)
                         {
-                            elementNames.Add(name ?? "Item" + j);
+                            elementNames.Add(name ?? TupleTypeSymbol.UnderlyingMemberName(j));
```

Comment:
It doesn't look like this line does the right thing. We already checked that name is not null. So we will simply fill this builder with the same name for all elements. Do we have a test for this code?